src/core_plugins/elasticsearch/lib/kibana_index_patches/patch_missing_title_keyword_fields.js

```diff
@@ -0,0 +1,68 @@
+import { getRootProperties } from '../../../../server/mappings';
+import { get } from 'lodash';
+import { KibanaIndexPatch } from './kibana_index_patch';
+
+const propertiesWithTitles = [
+  'index-pattern',
+  'dashboard',
+  'visualization',
+  'search',
+];
+
+export class PatchMissingTitleKeywordFields extends KibanaIndexPatch {
```

Comment:
Rather than subclasses of `KibanIndexPatch`, what if we just created instances of `KibanaIndexPatch` with configuration? I'm also not sure I see the value in separating `getUpdatedPatchMappings()` and `applyChanges()`

```js
export const missingTitleKeywordFieldsPath = new KibanaIndexPatch({
```